frontend/src/components/main-view/details/details-card/DetailsCard.tsx

```diff
@@ -33,11 +33,12 @@ function DetailsCard({fullScreen, handleClose}: Props) {
         login: <Login/>,
         employee: <EmployeeDetails/>,
         product: <ProductDetails/>,
-        customer: <div/>,
+        customer: <><div></div><></></>,
```

Comment:
not sure about this line. Do you need those empty fragments?

Reply:
no, don't need them. that's leftovers of a test